### PR TITLE
Adding items to cart

### PR DESCRIPTION
**Pull request by VinceCarollo**
## What does this PR do?

- Adds spec for adding cheese items to users cart
- Adds cheeses and merchants to seed file
- Adds bootstrap styling to items index page
- Adds create action in carts controller to pass items to cart PORO
- Adds Cart PORO to handle cart contents
- Adds button to items index to add items to cart
- Adds cart total next to cart link on nav bar

finishes #63

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -28,9 +28,16 @@
...
+            <li class="nav-item active">
+              <span class="nav-link" >
+                <%= link_to 'Cart', carts_path %>
+                <% if session[:cart] %>
+                  (<%= session[:cart].values.sum %>)
+                <% else %>
+                  (0)
+                <% end %>
+              </span>
+            </li>
```
Comment (BrennanAyers) [inline app/views/layouts/application.html.erb:40]: We can figure out the styling later, but this is great!

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,24 @@
...
+havarti = Item.create!(name: "Havarti", price: 68.99, image: 'http://eshfoods.com/media/catalog/product/cache/1/image/9df78eab33525d08d6e5fb8d27136e95/e/s/esh-058.jpg', inventory: 785, description: "Havarti or cream Havarti is a semisoft Danish cow's milk cheese. It is a table cheese that can be sliced, grilled, or melted.", user: merchant_2)
+mozerella = Item.create!(name: "Mozerella", price: 112.99, image: 'https://www.seriouseats.com/recipes/images/2015/10/20151017-pies-vicky-wasik-2.jpg', inventory: 48, description: "Mozzarella is a traditionally southern Italian cheese made from Italian buffalo's milk by the pasta filata method. Mozzarella received a Traditional Specialities Guaranteed certification from the European Union in 1998.", user: merchant_2)
```
Comment (BrennanAyers) [inline db/seeds.rb:21]: Bellisimo!

**Review by WHomer — APPROVED** [review]
nice work, looks sharp!

File: app/controllers/carts_controller.rb [hunk 3]
```diff
@@ -1,4 +1,13 @@
...
\ No newline at end of file
+
+  def create
+    item = Item.find(params[:item_id])
+    cart = Cart.new(session[:cart])
+    cart.add(item.id)
+    session[:cart] = cart.contents
+    flash.notice = "#{item.name} has been added to your cart"
+    redirect_to items_path
+  end
```
Comment (BrennanAyers) [inline app/controllers/carts_controller.rb:12]: Love it!

**Review by BrennanAyers — APPROVED** [review]

**Pull request merged**